src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -154,6 +156,26 @@ public static GroupTag parseGroupTag(String tag) throws ParseException {
         return new GroupTag(trimmedTag);
     }
 
+    /**
+     * Parses a {@code String tags} into a {@code String[]}
+     *
+     * @param tags String of tag to parse
+     * @return String[] of tags.
+     * @throws ParseException
+     */
+    public static String[] parseModuleMore(String tags) throws ParseException {
```

Comment:
Created another method to take in an Array of String.
Did this in a rush but in hindsight I could change this to ArrayList

Reply:
For naming conventions, `parserMoreModules` could have been a better name.

Reply:
Agreed. More clearer name as well.

Reply:
In addition, the method return type does not make sense according to the method name. It feels like it should return Modules instead of Strings.